pynest/nest/lib/hl_api_sonata.py

```diff
@@ -34,6 +33,13 @@
 from .hl_api_simulation import SetKernelStatus, Simulate
 from .hl_api_types import NodeCollection
 
+try:
+    import pandas as pd
+
```

Comment:
```suggestion
```

Reply:
Empty line after import is added by formatter.